package/hw8/User.java

```diff
@@ -0,0 +1,92 @@
+package hw8;
+
+public class User {
+
+    public String userName;
+    public String email;
+    public boolean isActive;
+    public double amountSpentMoney;
+
+    private String password;
+    private int age;
+
+    // password
+    public void setPassword(String userPassword) {
+        if (userPassword != null && !userPassword.isEmpty()) {
+            password = userPassword;
+        } else {
+            System.out.println("Incorrect password");
+        }
+    }
+    public String getPassword() {
+
+        return password;
+    }
+
+    // age
+    public void setAge(int userAge) {
+        if (userAge > 0) {
+            age = userAge;
+        } else {
+            System.out.println("Invalid age value");
+        }
+    }
+    public int getAge() {
+        return age;
+    }
+
+    // Метод makePurchase.
+    public void makePurchase(double purchasePrice) {
+        System.out.println("Purchase made for: " + purchasePrice);
+        amountSpentMoney += purchasePrice;
+    }
+    // Метод printTotalAmountOfSpentMoney.
+    public void printTotalAmountOfSpentMoney() {
+
+        System.out.println("Total amount of money spent: " + amountSpentMoney);
+    }
+
+    // 1. Дефолтний конструктор (без параметрів).
+    public User() {
```

Comment:
forgot to mentioned, that constructors within the class should be before methods and after fields